### DISCUSSION TITLE
Listening for new followers feature and modview.

### DISCUSSION THREAD

**Discussion by rodgms** [victornpb/twitch-mass-ban#1] [1. Questions & Answers] [answered]
Thanks a lot for the script!

I've noticed that the "listening for new followers" feature isn't working properly on my end, even with StreamElements/Streamlabs announcing the new followers in chat.

Also, I'd like to suggest having the hammer icon added to the modview as well, right now it only shows up in regular chat.

Cheers!

**Comment by victornpb** [comment, marked as answer]
are you using a customized message when users follow?
It currently relies on a message matching the following pattern
https://github.com/victornpb/twitch-mass-ban/blob/aa8985da071caed9ef462b4b14cf4267db1c12dc/twitchmassban.user.js#L309-L310

I can add other messages, if you send an example
